### DISCUSSION TITLE
run the repo locally

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [saadeghi/daisyui#1160] [Q&A] [answered]
I am trying to set up the repo locally and have to add `require("/path/to/dist/directory")` to `tailwind.config.js` but there are plenty of tailwind configs files. should I add it to all of them or what?
```js
module.exports = {
  plugins: [require("/path/to/dist/directory")],
};
```

**Comment by saadeghi** [comment, marked as answer]
You mean your project has plenty of config files?

`require("/path/to/dist/directory")]` means the path to the `/dist` directory of daisyUI repo. You just need to require it once in the `tailwind.config.js` of your own project.

**Reply by Ali-Hussein-dev** [reply]
I just want to try to add a component to the lib!

**Reply by saadeghi** [reply]
Start from [here](https://github.com/saadeghi/daisyui/blob/master/.github/CONTRIBUTING.md#to-run-the-documentation-site-on-local)  
Install dependencies and run the documentation site, then add new files.

**Reply by Ali-Hussein-dev** [reply]
I just followed the steps. but running `npm run dev` throw an error. I am not a Svelte dev, did I miss something?
![Screenshot 2022-09-21 213300](https://user-images.githubusercontent.com/42802922/191594358-a864db50-4063-4032-824a-ba5c81648bfd.jpg)

**Reply by saadeghi** [reply]
@Ali-Hussein-dev No you're not doing anything wrong. That's a weird error actually 🤔

I'm not sure but maybe updating Node would fix it.  
I also removed the mentioned *new lines* on that command so I would expect that version `2.31.0` fix that.

Can you try again after updating Node.js and getting daisyUI `2.31.0` to see if it works?

**Reply by Ali-Hussein-dev** [reply]
The error is gone. But a new one is popping up!

![Screenshot 2022-09-21 225507](https://user-images.githubusercontent.com/42802922/191608576-b3109a7e-0e36-49bd-b4c0-1599fa0b7098.jpg)

**Reply by saadeghi** [reply]
What's your Node.js version?  
These errors are just weird. I don't have windows to try but other contributors of daisyUI have been running the repo on local and there were never issues 😐

**Reply by Ali-Hussein-dev** [reply]
I updated it a few days agon and `v16.17.0`
